packages/nimble-components/src/utilities/style/types.ts

```diff
@@ -0,0 +1,5 @@
+export const ZIndexLevels = {
+    zIndex1: '1',
+    zIndex1000: '1000'
```

Comment:
Looking back at the original comment in https://github.com/ni/nimble/pull/530#discussion_r863076750, the reason this is 1000 is because it is a higher value than the [jqx grid](https://github.com/ni-kismet/jqx/blob/4b72d63749befd559576ecb258fbc6ad9a414b18/source/styles/default/jqx.table.css#L1033)

So we should include that context here

```suggestion
    // Original usages of 1000 were to compete with jqx-grid usage of z-index 200 for headers
    // See: https://github.com/ni/nimble/pull/530#discussion_r863076750
    zIndex1000: '1000'
```

Reply:
Ah, so that's where it was. I had searched for `z-index` being set to 200 somewhere and assumed it had been removed/changed since the comment was made. Didn't think to look in jqx code.